common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -232,18 +231,19 @@ public void run() {
                         } else {
                             Logger.info(
                                     TAG + methodName,
-                                    "Silent command result returned from cache."
+                                    "",
```

Comment:
Why are we passing an empty string here?

Reply:
This has to do with the way initially the Logger,info/infopii were coded. We dont always to print the entire RequestContext, it just adds to the clutter, hence adding a `""` because as it is we are logging the CorrelationId after the statement, which is essential to main the user readability.

Reply:
Made changes to the log statements, and changed this part, to not pass the empty string.